pkgs/development/libraries/boxfort/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, coreutils, fetchFromGitHub, cmake, meson, ninja, pkg-config, python3, git }:
+
+stdenv.mkDerivation rec {
+  pname = "boxfort-unstable";
+  version = "2019-09-19";
```

the pname should just be the name of the package, unstable should go into version
```suggestion
  pname = "boxfort";
  version = "unstable-2019-09-19";
```

---

I thought @worldofpeace and @jtojnar were recently advocating for the `unstable` bit to go in `name`... :thinking:

---

@aanderse The thing is `unstable` ends up in `name` anyway when it gets split.
ref https://github.com/NixOS/nixpkgs/issues/68531#issuecomment-533760929

---

My bad. Thanks for clarifying @worldofpeace!

---

so does it still stand?

---

Nope, it's resolved.

---

still seems odd to me for the package "boxfort" to become "boxfort-unstable" for a given commit, especially if we wanted to reference "${pname}" in the expression.

---

the linked issue is a better place for discussion than this comment, so I'll stop my rant :)